Automattic/studio
# Add Windows Release Builds

## Proposed Changes
Creates signed Windows builds each time a new tag is created.

## Testing Instructions
See https://a8c.slack.com/archives/C06DRMD6VPZ/p1715362399654439, note that each build is downloadable (and signed)
